src/formaction.cpp

```diff
@@ -259,6 +260,26 @@ std::vector<std::string> FormAction::get_suggestions(
 							line);
 					}
 				}
+			} else if (tokens[0] == "exec") {
+				if (tokens.size() <= 2) {
+					std::vector<KeyMapDesc> descs;
+					v->get_keymap()->get_keymap_descriptions(
+						descs,
+						v->get_keymap()->get_flag_from_context(this->id())
```

Comment:
#897 made `get_flag_from_context()` private. Please rebase onto current master and make the method public again to fix compilation errors here.